cmake/Findquatlib.cmake

```diff
@@ -101,4 +117,7 @@ else()
 	set(QUATLIB_INCLUDE_DIRS)
 endif()
 
-mark_as_advanced(QUATLIB_LIBRARY QUATLIB_INCLUDE_DIR)
+mark_as_advanced(QUATLIB_LIBRARY_RELEASE
+				 QUATLIB_LIBRARY_DEBUG
```

Comment:
On GitHub, this looks way more indented than the rest, which makes me suspect it's using tabs and the rest of the file is using spaces. I'm not super-addicted to a single convention over another, but consistency within a file improves readability. Just double-check (preferably in an editor with a "show whitespace" feature - many have this, on Windows, Notepad2-mod is great for whitespace cleanup) that your changes are internally consistent with the existing code in the file.